tests/test_renderer.py

```diff
@@ -1,10 +1,25 @@
 import pytest
 
 import pyvista
+from pyvista import _vtk
```

Comment:
Almost all our tests use `import vtk`, treating the test as external to the library. I think we should do that here too (and probably fix up the other outliers eventually).
```suggestion
import vtk
```